sympy/functions/combinatorial/factorials.py

```diff
@@ -288,7 +288,34 @@ def _eval_as_leading_term(self, x):
         return self.func(arg)
 
 class MultiFactorial(CombinatorialFunction):
-    pass
+
+    @classmethod
+    def eval(cls, n, alpha):
+        if not alpha.is_integer or not alpha.is_positive:
+            raise ValueError("alpha should be a positive integer")
+        if alpha == 1:
+            return factorial(n)
+        if alpha == 2:
+            factorial2(n)
+
+        if n.is_Number and alpha.is_Number:
+            if not n.is_Integer:
+                raise ValueError("argument should be non negative or negative "
+                                     "argument should not be divisible by alpha")
+            if n <= 0 and n > -alpha:
```

Comment:
Does this work, if n is `Symbol`?

Reply:
@czgdp1807 It should not work  as `n % alpha` condition will  cause complications.

Reply:
So what will happen if I do,
```python
>>> MultiFactorial(n, k)
```